plugins/user-settings/src/components/FeatureFlags/FeatureFlagsItem.tsx

```diff
@@ -30,16 +30,20 @@ type Props = {
   toggleHandler: Function;
 };
 
+const getSecondaryText = (flag: FeatureFlag) => {
+  if (flag.description) {
+    return flag.description;
+  }
+  return flag.pluginId ? `Registered in ${flag.pluginId} plugin` : undefined;
```

Comment:
Maybe instead of returning `undefined` you can return a default string like `Registered`?

Reply:
Sure but does that indicate anything really? Maybe `Registered in the application` or something?